libs/application/templates/estate/src/lib/constants.ts

```diff
@@ -36,3 +36,12 @@ export const EstateTypes = {
   permitForUndividedEstate: 'Seta í óskiptu búi',
   divisionOfEstateByHeirs: 'Einkaskipti',
 }
+
+export const relationWithApplicant = [
+  'Barn (sameiginlegt barn hjóna)',
+  'Stjúpbarn (barn látins maka)',
+  'Barnabarn (sameiginlegt barnabarn hjóna)',
+  'Stjúpbarnabarn (barnabarn látins maka)',
+  'Barnabarnabarn (sameiginlegt barnabarnabarn hjóna)',
+  'Stjúpbarnabarnabarn (barnabarnabarn látins maka)',
+]
```

Comment:
Can we add a comment about why the list is here?

```suggestion
// The District Commissioner's relation endpoint was not suitable for this list.
// Another endpoint won't be created for this static list.
// This is here because it is specific for this application and is not subject to change.
export const relationWithApplicant = [
  'Barn (sameiginlegt barn hjóna)',
  'Stjúpbarn (barn látins maka)',
  'Barnabarn (sameiginlegt barnabarn hjóna)',
  'Stjúpbarnabarn (barnabarn látins maka)',
  'Barnabarnabarn (sameiginlegt barnabarnabarn hjóna)',
  'Stjúpbarnabarnabarn (barnabarnabarn látins maka)',
]
```

Reply:
i think ill double check with the list again